addons/account/report/account_aged_partner_balance.py

```diff
@@ -47,7 +47,7 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
         cr = self.env.cr
         user_company = self.env.company
         user_currency = user_company.currency_id
-        company_ids = self._context.get('company_ids') or [user_company.id]
+        company_ids = self.env.companies.ids
```

Comment:
if 'allowed_company_ids' in context : self.env.companies else self.env.company maybe